test/e2e/tests/confirmations/transactions/increase-token-allowance-redesign.spec.ts

```diff
@@ -0,0 +1,226 @@
+import FixtureBuilder from '../../../fixture-builder';
+import {
+  defaultGanacheOptions,
+  defaultGanacheOptionsForType2Transactions,
+  largeDelayMs,
+  veryLargeDelayMs,
+  WINDOW_TITLES,
+  withFixtures,
+} from '../../../helpers';
+import { Mockttp } from '../../../mock-e2e';
+import GanacheContractAddressRegistry from '../../../seeder/ganache-contract-address-registry';
+import { SMART_CONTRACTS } from '../../../seeder/smart-contracts';
+import { Driver } from '../../../webdriver/driver';
+import { scrollAndConfirmAndAssertConfirm } from '../helpers';
+import { openDAppWithContract, TestSuiteArguments } from './shared';
+
+describe('Confirmation Redesign ERC20 Increase Allowance', function () {
+  const smartContract = SMART_CONTRACTS.HST;
+
+  describe('Submit an increase allowance transaction @no-mmi', function () {
+    it('Sends a type 0 transaction (Legacy) with a small spending cap', async function () {
+      await withFixtures(
+        {
+          dapp: true,
+          fixtures: new FixtureBuilder()
+            .withPermissionControllerConnectedToTestDapp()
+            .withPreferencesController({
+              preferences: {
+                redesignedConfirmationsEnabled: true,
+                isRedesignedConfirmationsDeveloperEnabled: true,
+              },
+            })
+            .build(),
+          ganacheOptions: defaultGanacheOptions,
+          smartContract,
+          testSpecificMock: mocks,
+          title: this.test?.fullTitle(),
+        },
```

Comment:
[suggestion] I think we could DRY here more by creating a function to return the fixture options

```suggestion
        generateFixtureOptions({ ganacheOptions: isRedesignedConfirmationsDeveloperEnabled})
```

where the function could be something like
```
function createFixtureOptions(options = {}): object {
  return {
    dapp: true,
    fixtures: new FixtureBuilder()
      .withPermissionControllerConnectedToTestDapp()
      .withPreferencesController({
        preferences: {
          redesignedConfirmationsEnabled: true,
          isRedesignedConfirmationsDeveloperEnabled: true,
        },
      })
      .build(),
    ganacheOptions: defaultGanacheOptions,
    smartContract,
    testSpecificMock: mocks,
    title: this.test?.fullTitle(),
    ...options,
  };
}
```